src/serializer.ts

```diff
@@ -29,22 +29,11 @@ class Serializer {
     transaction: Transaction
   ): object | undefined {
     // Serialize the protocol buffer to a JSON representation.
-    var object: any = transaction.toObject();
-
-    // Convert fields names where direct conversion is possible.
-    this.convertPropertyName("sequence", "Sequence", object);
-    this.convertPropertyName("signingPublicKey", "SigningPubKey", object);
-    this.convertPropertyName(
-      "lastLedgerSequence",
-      "LastLedgerSequence",
-      object
-    );
-
-    // Delete unsupported fields from the protocol buffer.
-    ['memosList', 'flags', 'signature', 'signersList', 'sourceTag', 'accountTransactionId'].forEach((key): boolean => delete object[key]);
+    var object: any = {}
```

Comment:
Replace `var` and could we type this object now?

Reply:
Done.